content/javascript/concepts/arrays/terms/filter/filter.md

```diff
@@ -0,0 +1,119 @@
+---
+Title: ".filter()"
+Description: "Creates a new array with all elements that pass the test implemented by the provided function."
+Subjects:
+  - "Web Development"
+  - "Computer Science"
+Tags:
+  - "Arrays"
+  - "Methods"
+CatalogContent:
+  - "introduction-to-javascript"
+  - "paths/front-end-engineer-career-path"
+---
+
+The `.filter()` array method creates a new array with all elements that pass the test implemented by the provided function.
+
+
+## Syntax
+
+```js
+// Arrow function
+array.filter(element => { ... })
+array.filter((element, index) => { ... })
+array.filter((element, index, array) => { ... })
+```
+
+## Parameters
+
+*element*
+    The current element being processed in the array.
+
+*index* (optional)
+    The index of the current element being processed in the array.
+
+*array* (optional)
+    The array *filter* was called upon.
+
+## Return Value
+A new array with the elements that pass the test. 
+
+note: **If no elements that pass the test, an empty array will be returned.**
```

Comment:
Bold the note, and not the note text and minor tweaks:

**Note:** If no elements pass the test, an empty array will be returned.